sw.js

```diff
@@ -0,0 +1,46 @@
+var CACHE_NAME = 'IESN_IG_DA';
+var urlsToCache = [
+    '/da/',
+    '/da/styles/app.css',
+    '/da/styles/pcp-highlight.css',
+    '/da/styles/print.css',
+    '/da/styles/reset.css',
+    '/da/styles/toastr.min.css',
+    '/da/scripts/app.js',
+    '/da/scripts/PseudoCodeParser.js',
+    '/da/scripts/libraries/jquery-3.2.1.min.js',
+    '/da/scripts/libraries/toastr.min.js',
+    'https://use.fontawesome.com/ca53c7c19d.css',
+    'https://use.fontawesome.com/ca53c7c19d.js',
+];
+
+self.addEventListener('install', function(event) {
+    // Perform install steps
+    event.waitUntil(
+        caches.open(CACHE_NAME)
+        .then(function(cache) {
+            return cache.addAll(urlsToCache);
+        })
+    );
+});
+
+self.addEventListener('activate', function(event) {
+    console.log('Service worker is activated!');
```

Comment:
[nice to have] j'utiliserais `console.debug` au lieu d'un `console.log` dans ce cas-ci.